packages/yoastseo/src/scoring/assessments/seo/ProductSKUAssessment.js

```diff
@@ -79,7 +99,9 @@ export default class ProductSKUAssessment extends Assessment {
 	 */
 	scoreProductSKU( productSKUData, config ) {
 		// If a product has no variants, return orange bullet if it has no global SKU, and green bullet if it has one.
```

Comment:
This description might also not relevant anymore

Reply:
agreed